Description: fix typo
 Fix typo
 .
 toped (0.0+r2361-1) unstable; urgency=low
 .
   * Initial release (Closes: #692561)
Author: Xiangfu Liu <xiangfu@openmobilefree.net>
Bug-Debian: http://bugs.debian.org/692561

--- /dev/null
+++ toped-0.0+r2361/fix-typo.patch
@@ -0,0 +1,41 @@
+Index: tpd_ifaces/oasis_io.cpp
+===================================================================
+--- tpd_ifaces/oasis_io.cpp	(revision 2361)
++++ tpd_ifaces/oasis_io.cpp	(working copy)
+@@ -1836,7 +1836,7 @@
+ {
+    byte scheme = ofn.getByte();
+    if (scheme & 0xF0)
+-      ofn.exception("Bad extention type (27.? - not explicitly ruled-out)");
++      ofn.exception("Bad extension type (27.? - not explicitly ruled-out)");
+    // deal with the start extenstion
+    byte extype = (scheme & 0x0c) >> 2;
+    if (0 != extype)
+Index: tpd_ifaces/oasis_io.h
+===================================================================
+--- tpd_ifaces/oasis_io.h	(revision 2361)
++++ tpd_ifaces/oasis_io.h	(working copy)
+@@ -120,8 +120,8 @@
+    /*! The enum values correspond to the Path Extenstion Shemes as defined in the
+        standard for SS and EE bits (27.8, Table 15) The last member ex_unknown is
+        added to maintain proper state in the Oasis::PathExtensions which in turn
+-       is used as a base class of Cell::_mod_exs /path-start-extention/ and
+-       Cell::_mod_exe /path-end-extention/ modal variables
++       is used as a base class of Cell::_mod_exs /path-start-extension/ and
++       Cell::_mod_exe /path-end-extension/ modal variables
+    */
+    typedef enum { ex_reuse         = 0 ,
+                   ex_flush         = 1 ,
+Index: tpd_DB/tedat_ext.h
+===================================================================
+--- tpd_DB/tedat_ext.h	(revision 2361)
++++ tpd_DB/tedat_ext.h	(working copy)
+@@ -16,7 +16,7 @@
+ //           $URL$
+ //        Created: Sun Aug 9 2010
+ //     Originator: Sergey Gaitukevich - gaitukevich.s@toped.org.uk
+-//    Description: Layout primitives with extentions
++//    Description: Layout primitives with extensions
+ //---------------------------------------------------------------------------
+ //  Revision info
+ //---------------------------------------------------------------------------
--- toped-0.0+r2361.orig/tpd_DB/tedat_ext.h
+++ toped-0.0+r2361/tpd_DB/tedat_ext.h
@@ -16,7 +16,7 @@
 //           $URL: http://toped.googlecode.com/svn/trunk/tpd_DB/tedat_ext.h $
 //        Created: Sun Aug 9 2010
 //     Originator: Sergey Gaitukevich - gaitukevich.s@toped.org.uk
-//    Description: Layout primitives with extentions
+//    Description: Layout primitives with extensions
 //---------------------------------------------------------------------------
 //  Revision info
 //---------------------------------------------------------------------------
--- toped-0.0+r2361.orig/tpd_ifaces/oasis_io.cpp
+++ toped-0.0+r2361/tpd_ifaces/oasis_io.cpp
@@ -1836,7 +1836,7 @@ void Oasis::Cell::readExtensions(OasisIn
 {
    byte scheme = ofn.getByte();
    if (scheme & 0xF0)
-      ofn.exception("Bad extention type (27.? - not explicitly ruled-out)");
+      ofn.exception("Bad extension type (27.? - not explicitly ruled-out)");
    // deal with the start extenstion
    byte extype = (scheme & 0x0c) >> 2;
    if (0 != extype)
--- toped-0.0+r2361.orig/tpd_ifaces/oasis_io.h
+++ toped-0.0+r2361/tpd_ifaces/oasis_io.h
@@ -120,8 +120,8 @@ namespace Oasis {
    /*! The enum values correspond to the Path Extenstion Shemes as defined in the
        standard for SS and EE bits (27.8, Table 15) The last member ex_unknown is
        added to maintain proper state in the Oasis::PathExtensions which in turn
-       is used as a base class of Cell::_mod_exs /path-start-extention/ and
-       Cell::_mod_exe /path-end-extention/ modal variables
+       is used as a base class of Cell::_mod_exs /path-start-extension/ and
+       Cell::_mod_exe /path-end-extension/ modal variables
    */
    typedef enum { ex_reuse         = 0 ,
                   ex_flush         = 1 ,
